map.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Bushfire Intensity</title>
+
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.3/dist/leaflet.css" integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ==" crossorigin="" />
+    <script src="https://unpkg.com/leaflet@1.3.3/dist/leaflet.js" integrity="sha512-tAGcCfR4Sc5ZP5ZoVz0quoZDYX5aCtEm/eu1KhSLj2c9eFrylXZknQYmxUssFaVJKvvc0dJQixhGjG2yXWiV9Q==" crossorigin=""></script>
+    <script src="https://d3js.org/d3.v5.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
+    <div id="map"></div>
+
+
+    <!-- API key -->
+    <script type="text/javascript" src="static/js/config.js"></script>
```

Comment:
I see you put config.js in the .gitignore, which prevents the leaflet api key from being uploaded to github. That's good. But, I _**think(but not sure??)**_ this still exposes the api key in the underlying source code. When the code is deployed, if someone did a view page source or an inspect on our page, could they click into "static/js/config.js"?

Reply:
I am not sure the best way to handle environment variables/api keys like this. I wonder if there is a way to safely put the key in the flask server on the backend and then pass it down to the js somehow?? Just speaking randomly out loud.

Reply:
You bring up a good point! I was wondering myself about this. I'll raise in office hours tonight with Paul et al.

Reply:
Paul has confirmed - we'll handle these credentials via Heroku. For now, I'll leave it in so the map still functions, and we can replace when we get credentials working via Heroku.